.gitignore

```diff
@@ -4,7 +4,7 @@
 
 # Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
 # More information here https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html
-Cargo.lock
+#Cargo.lock
```

Comment:
@acevif 

The .gitignore file uses [the GitHub defaults](https://github.com/github/gitignore/blob/main/Rust.gitignore).

> Remove Cargo.lock from gitignore if creating an executable, leave it for libraries

As it says, I see no problem if the author of the executable himself removes it.

If Cargo.lock is to be commented out, I would like to see the comments changed for the library authors.